CMakeLists.txt

```diff
@@ -113,12 +114,13 @@ add_dependencies(${TELEPATHY_OFONO} schema_update qrc_update)
 
 enable_testing()
```

Comment:
Might it also be worth it to add regression tests for these changes?

Reply:
There are not QPulseAudioEngine tests yet. What would be needed, more than just headset detection?

Reply:
Basically anything that does anything should be tested to ensure it does what it reports itself to do... But if there are no tests on this module at all I suspect it was determined to be too difficult or its logic was too trivial. Does it seem to call out to other sources to do anything, or do you think it's relatively self-contained?

Reply:
QPulseAudioEngine communicates with PulseAudio which might be hard to test thoroughly.

Reply:
Okay... *hangs head in no-test shame*